lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AccessTokenService.java

```diff
@@ -27,18 +41,30 @@
     public static final String REDIRECT_URI = "redirect_uri";
     public static final String GRANT_TYPE = "grant_type";
     public static final String CLIENT_ID = "client_id";
+    public static final String CLIENT_ASSERTION_TYPE = "client_assertion_type";
+    public static final String CLIENT_ASSERTION = "client_assertion";
     private final DataStore<AddressSessionItem> dataStore;
     private final Long bearAccessTokenTtl;
-
-    public AccessTokenService(DataStore<AddressSessionItem> dataStore, Long bearerAccessTokenTtl) {
+    private final ConfigurationService configurationService;
+    private final JWTVerifier jwtVerifier;
+
+    public AccessTokenService(
+            DataStore<AddressSessionItem> dataStore,
+            Long bearerAccessTokenTtl,
+            ConfigurationService configurationService,
+            JWTVerifier jwtVerifier) {
```

Comment:
added JWTVerifier as its now used in the AddressSessionService and AddressTokenService

Reply:
added unit tests for JWTVerifier.class